frontend-opal/src/components/PortfolioContainer/LineChart/LineChart.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { View } from 'react-native';
+import { VictoryLine, VictoryChart, VictoryAxis, VictoryScatter } from 'victory-native';
+
+import dataSample from './dataSample';
+
+const InstrumentDetail = () => {
+  const formatAxisMonth = date => {
+    const options = { month: 'short' };
+    return new Intl.DateTimeFormat('en-US', options).format(date);
```

Comment:
@Yuan-EPAM let's just use `date.getMonth()` for now

Reply:
snapshot tests have been added~